app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -4,12 +4,35 @@ class SatisfactionSurveyForm
 
     attr_accessor :question, :answer
 
+    QUESTIONS_WE_ASK = [
+      'I would recommend this service to a friend or colleague',
+      'I found this service unnecessarily complex',
```

Comment:
Should we pick these up from the titles?

Reply:
💯